install.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# vim: tabstop=4 shiftwidth=4 softtabstop=4
+# -*- sh-basic-offset: 4 -*-
+
+set -euo pipefail
+
+if [[ $(id -u) -ne 0 ]]; then
+    echo "Please run as root or using sudo"
+    exit 1
+fi
+
+mkdir -p /opt/wott
+
+if [ ! -f /etc/wott/config.ini ]; then
+    echo -e "[DEFAULT]\\nenroll_token = $CLAIM_TOKEN" > /opt/wott/config.ini
```

Comment:
You're checking for `/etc/wott/config.ini` and then writing to `/opt/wott/config.ini`. Also where is `CLAIM_TOKEN` supposed to come from?

Reply:
I'm checking to make sure it doesn't exist - and only if it *doesn't* exist, i write to it.

Claim token comes from an environment variable as mentioned in Slack.

Reply:
Please read again. You're checking for `/etc/wott/config.ini` but writing to `/opt/wott/config.ini`.

Reply:
This is why we do peer reviews :)